app/views/common_parameters/_parameters_table.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="tab-pane" id="params">
+  <div class="tab-content">
+    <ul class="nav nav-tabs" data-tabs="tabs">
```

Comment:
we opted for a less confusing UI and the puppetclass/global tabs would have a different style so it won't look the same as the host edit tabs.